requirements.txt

```diff
@@ -11,4 +11,6 @@ requests==2.24.0
 requests-mock==1.8.0
 social-auth-app-django==4.0.0
 social-auth-core==3.3.3
-whitenoise==5.2.0
\ No newline at end of file
+whitenoise==5.2.0
+psycopg2-binary==2.8.6
```

Comment:
Please remove the psycopg2-binary. That should only appear in requirements-prod.txt not here. Basically a local developer installs the requirements from requirements.txt and requirements-dev.txt. On deployment and in travis we install requirements from requirements.txt and requirements-prod.txt

Reply:
fixed